Makefile.versions

```diff
@@ -1,7 +1,7 @@
 # Calculate versions; these can be overridden
 CUTTING_EDGE := devel
 DEV_VERSION := dev
-override CALCULATED_VERSION := $(shell git describe --tags --dirty="-$(DEV_VERSION)" --exclude="*$(CUTTING_EDGE)*" --exclude="latest")
+override CALCULATED_VERSION := $(BASE_BRANCH)-$(shell git rev-parse --short HEAD)$(shell test -z `git diff-index HEAD` || echo "-$(DEV_VERSION)")
```

Comment:
We need to handle tags to, when we are on 0.9.0, 0.9.1, etc... so we can publish $image:0.9.0, $image:0.9.1

Or am I missing something?

Reply:
Yes I'm missing that we do that now on the releases repository ':D (tagging the final versions...)